src/test/functional/features/applicant2/how-did-you-change-your-name.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Applicant 2 how did you change your name
+
+  Background:
+    Given I login
+    And I am reviewing an application for divorce created by my wife
+    When I go to '/applicant2/how-did-you-change-your-name'
+    Then the page should include "How did you change your name?"
+
+  Scenario: They changed their name by sending of their certificate
+    Given I select "By sending off my marriage certificate"
+    When I click "Continue"
+    Then the page should include "How the court will contact you"
+
+  Scenario: They changed their name by deed poll or statutory declaration
+    Given I select "By deed poll or ‘statutory declaration’"
+    When I click "Continue"
+    Then the page should include "How the court will contact you"
+
+  @nightly
+  Scenario: They did not indicate how their name was changed
+    Given I select "Another way"
+    When I click "Continue"
+    Then the page should include "There was a problem"
```

Comment:
if its because the page isnt implemented, you can use
`Then the page URL should be "/URL-HERE"`

Reply:
thanks but it should fail because of a text box that isn't filled in